src/spec.tsx

```diff
@@ -1,51 +1,97 @@
-import * as React from "react";
+import React, { useState } from "react";
 import { render, cleanup, fireEvent } from "@testing-library/react";
 
 import { KeyHandler, Provider, FocusGroup } from "./index";
 
-describe("<KeyHandler />", () => {
-  afterEach(cleanup);
-
-  test("matching key is triggered", () => {
-    const handlerSpy = jest.fn();
-    const anotherSpy = jest.fn();
-    const shadowedGroupSpy = jest.fn();
-    const unusedHandlerSpy = jest.fn();
-    render(
+function TestComponent() {
+  const [clickCounts, setClickCounts] = useState<Record<string, number>>({
+    parent: 0,
+    child: 0,
+    sibling: 0,
+    unused: 0,
+  });
+  return (
+    <div>
       <div>
-        <Provider>
+        <span>Parent: {clickCounts.parent}</span>
+        <span>Child: {clickCounts.child}</span>
+        <span>Sibling: {clickCounts.sibling}</span>
+        <span>Unused: {clickCounts.unused}</span>
```

Comment:
What a creative way to test this! 🥇 So clear!

Reply:
Yeah, I needed a way to test interactively showing and hiding a group, and this seemed like the easiest way to do it 😆